AudreyW_Lab_1.ipynb

```diff
@@ -0,0 +1,598 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.distplot(chix)  \n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.15729920705028488\n",
+      "0.11384629800665769\n"
+     ]
+    }
+   ],
+   "source": [
+    "sigma_test3 = 1-stats.chi2.cdf(2, df=1)\n",
```

Comment:
I think before you start making these calculations, be sure to add in the statistical question you are trying to answer, and convert that question to an integral.